app/import.go

```diff
@@ -196,11 +196,11 @@ func (a *App) bulkImport(c *request.Context, jsonlReader io.Reader, attachmentsR
 
 		var line LineImportData
 		if err := decoder.Decode(&line); err != nil {
-			return model.NewAppError("BulkImport", "app.import.bulk_import.json_decode.error", nil, err.Error(), http.StatusBadRequest), lineNumber
+			return model.NewAppError("BulkImport", "app.import.bulk_import.json_decode.error", nil, "", http.StatusBadRequest).Wrap(err), lineNumber
 		}
 
 		if err := processAttachments(&line, importPath, attachedFiles); err != nil {
-			return model.NewAppError("BulkImport", "app.import.bulk_import.process_attachments.error", nil, err.Error(), http.StatusBadRequest), lineNumber
+			c.Logger().Warn("Error while processing import attachments. Objects might be broken.", mlog.Err(err))
```

Comment:
Is this relevant? Maybe you need to update the branch..

Reply:
To be honest I don't exactly know why it's displaying this as a change. I'll do an update-branch.